src/os/bluestore/BlueStore.h

```diff
@@ -299,39 +299,29 @@ class BlueStore : public ObjectStore,
     uint16_t state;             ///< STATE_*
     uint16_t cache_private = 0; ///< opaque (to us) value used by Cache impl
     uint32_t flags;             ///< FLAG_*
-    uint64_t seq;
```

Comment:
How about leaving `seq` field, but limiting its use to dout?
I remember being able to track seq numbers was *very* helpful in debugging.

Reply:
Reverting seq is not a big deal but why not using txc pointer for such a tracking instead? Generally they're more unique most of the time than seq values which start from 0 and hence have much more unrelated matchings.